msteams-platform/bots/calls-and-meetings/calls-meetings-bots-overview.md

```diff
@@ -97,11 +109,9 @@ Real-time media refers to scenarios where media must be processed in real-time,
 ## See also
 
 * [Build bots for Teams](../what-are-bots.md)
-* [Work with the cloud communications API in Microsoft Graph](/graph/api/resources/communications-api-overview)
+* [Microsoft Graph SDK overview](/graph/sdks/sdks-overview)
 * [Add Graph permissions](registering-calling-bot.md#add-graph-permissions)
-* [Develop calling and online meeting bots on your local PC](debugging-local-testing-calling-meeting-bots.md)
```

Comment:
Question for this and the next line removed, those docs still exist, do we not need to link them anymore? Are they not relevant?

Reply:
@samarthasthana, these links either exist in the article or appear among the list of links in the `Documentation` section (on the right-hand side of the screen). Hence, I have removed them.